packages/integration_test/example/android/app/build.gradle

```diff
@@ -39,7 +39,7 @@ android {
     defaultConfig {
         // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
         applicationId "com.example.integration_test_example"
-        minSdkVersion 16
+        minSdkVersion 19
```

Comment:
nit: it'd be helpful to explain why this API level is chosen here.

Reply:
reverted.